OpenRA.Mods.Common/AI/HackyAI.cs

```diff
@@ -280,6 +278,7 @@ public CPos GetRandomBaseCenter()
 		BitArray resourceTypeIndices;
 
 		AIHarvesterManager harvManager;
+		AISupportPowerManager supportPowerManager;
```

Comment:
Calling this `supportPowerManager` is asking for trouble.  This class already has a `SupportPowerManager supportPowerMngr`!

Can you please rename `supportPowerMngr` to `supportPowerManager` and then this to something else?

Reply:
Or, better, move `supportPowerMngr` inside the new class (but also fix the variable naming)